deps/jerry/jerry-core/jmem/jmem-heap.c

```diff
@@ -154,7 +154,7 @@ jmem_heap_init (void)
 #ifndef JERRY_SYSTEM_ALLOCATOR
   JERRY_ASSERT ((uintptr_t) JERRY_HEAP_CONTEXT (area) % JMEM_ALIGNMENT == 0);
 
-  JERRY_CONTEXT (jmem_heap_limit) = CONFIG_MEM_HEAP_DESIRED_LIMIT;
```

Comment:
Where was `CONFIG_MEM_HEAP_DESIRED_LIMIT` defined? Would it be used anymore?

Reply:
CONFIG_MEM_HEAP_DESIRED_LIMIT default value is 8192 (config.h). Beside this place will be use in jmem_heap_stat_alloc and() and jmem_heap_stat_free() function. This change could not effect anything else.

Reply:
Why not increase `CONFIG_MEM_HEAP_DESIRED_LIMIT` instead of adding another definition of whatever *LIMIT_VALUE*.